### ISSUE TITLE
mauve compilation errors (when trying to run Java11 tests)

### ISSUE THREAD

**Issue by smlambert** [AdoptOpenJDK/openjdk-systemtest#148]
Not sure if I am missing a prereq (doesn't appear that any setup step fails), but failing to compile when attempting to run some Java11 tests:

23:03:20     [javac] /tmp/mauve/mauve/gnu/testlet/org/omg/CORBA_2_3/ORB/Valtype/cmInfo.java:15: error: package org.omg.CORBA does not exist
23:03:20     [javac] import org.omg.CORBA.DataInputStream;
23:03:20     [javac]                     ^
23:03:20     [javac] /tmp/mauve/mauve/gnu/testlet/org/omg/CORBA_2_3/ORB/Valtype/cmInfo.java:16: error: package org.omg.CORBA does not exist
23:03:20     [javac] import org.omg.CORBA.DataOutputStream;
23:03:20     [javac]                     ^

...

23:03:21     [javac]   symbol:   class BAD_INV_ORDER
23:03:21     [javac]   location: class testAny
23:03:21     [javac] /tmp/mauve/mauve/gnu/testlet/org/omg/CORBA/Any/testAny.java:111: error: cannot find symbol
23:03:21     [javac]                             throws BAD_OPERATION
23:03:21     [javac]                                    ^

23:03:33     [javac] 100 errors
23:03:33     [javac] 46 warnings

**Comment by pshipton** [comment]
CORBA is one of the packages removed from Java 11.
https://github.com/eclipse/openj9/issues/992

**Comment by smlambert** [comment]
Thanks @pshipton that explains it!  

@Mesbah-Alam @lumpfish - Can someone look into this and to any other changes required in systemtest for Java11 support?  There are already some [Java11 builds ready for testing against
](https://ci.adoptopenjdk.net/view/JDK11%20WIP/)...

**Comment by Mesbah-Alam** [comment]
Mauve is compiled and used as the workload in the load tests. We need to add an "EE implementation from elsewhere, such as Eclipse Jakarta", for Mauve to compile on Java 11. I will try it out locally and update this PR.

**Comment by lumpfish** [comment]
I would advocate removing the tests for the removed components.  There are already lots of tests which are excluded for one reason or another - the tests' requirement for prereqs being one of them - see https://github.com/AdoptOpenJDK/openjdk-systemtest/blob/master/openjdk.test.mauve/build.xml for how they are excluded at build (configure) time.  The tests would also need removing from the load test inventory files in https://github.com/AdoptOpenJDK/openjdk-systemtest/tree/master/openjdk.test.load/config/inventories/mauve

**Comment by lumpfish** [comment]
@Mesbah-Alam - do you know which are the tests we need to remove or do we need to do some local testing to get the list?

**Comment by Mesbah-Alam** [comment]
@lumpfish - I have removed the CORBA folders and subsequently been removing each referenced Java files (and Java files which fail to compile due to the removal of them) incrementally. Once I reach the point where no Java file needs to be removed I'l run the tests locally to see if they still run with these removals in place. I didn't see avy direct references of any CORBA classes in any of the inventory files-- but there could be references of tests that have been removed, in which case they'd have to be removed from the inventory xml too.

**Comment by Mesbah-Alam** [comment]
Peter Shipton confirmed, `java.transaction` module, on which implementation of several modularity tests depend, have been completely removed in Java 11 as part of https://github.com/eclipse/openj9/issues/992

All the following modules have been removed in java 11. 
   
 ```
java.xml.ws (JAX-WS, plus the related technologies SAAJ and Web Services Metadata)
    java.xml.bind (JAXB)
    java.activation (JAF)
    java.xml.ws.annotation (Common Annotations)
    java.corba (CORBA)
    java.transaction (JTA)

    java.se.ee (Aggregator module for the six modules above)
    jdk.xml.ws (Tools for JAX-WS)
    jdk.xml.bind (Tools for JAXB
```

This means we need to re-implemented using modules that exist in Java 11.  Tests that need to be rewritten: 
```
1) /openjdk.test.modularity/src/tests/com.test/net/adoptopenjdk/test/modularity/junit/TestStressLayers.java
2) /openjdk.test.modularity/src/com.stf/net/adoptopenjdk/test/modularity/PatchModuleImageTest.java
3) /openjdk.test.modularity/src/com.stf/net/adoptopenjdk/test/modularity/PatchModuleTest.java
4) /openjdk.test.modularity/src/com.stf/net/adoptopenjdk/test/modularity/UpgradeModPathTest.java
```

**Comment by Mesbah-Alam** [comment]
Systemtests compile fine with test.modularity excluded: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/Grinder/106/console

I am going to create a PR that contains deletion of the corba related mauve classes and exclusion for modularity for Java 11.

**Comment by Mesbah-Alam** [comment]
Note: this only unblocks the Java 11 build to run JCK tests which are of higher priority. The personal build I ran didn't run any systemtests as I need to update the playlist and add `<subset>SE110</subset>` to the tests which will be handled in a separate PR.
